client/order/index.js

```diff
@@ -17,6 +17,7 @@ import { formatExplicitCurrency } from 'utils/currency';
 import { reasons } from 'wcpay/disputes/strings';
 import { getDetailsURL } from 'wcpay/components/details-link';
 import { disputeAwaitingResponseStatuses } from 'wcpay/disputes/filters/config';
```

Comment:
Not for this PR, but makes me think we could improve `disputeAwaitingResponseStatuses` logic by creating an `isAwaitingResponse` util function, similar to `isInquiry`.